test/rxd/correct_data/ecs_example_tort.dat

```diff
@@ -0,0 +1 @@
+ecs_example.dat
```

Comment:
if everything is correct... shouldn't this pull request make no changes to anything except 2, 1, or 0 dimensional extracellular simulations? I'm worried about that dt issue from above.